casepro/pods/base.py

```diff
@@ -0,0 +1,65 @@
+import json
+from confmodel import fields, Config as ConfmodelConfig
+from django.apps import AppConfig
+
+
+class PodConfig(ConfmodelConfig):
+    '''
```

Comment:
Resolved in https://github.com/rapidpro/casepro/pull/107/commits/327c985ee544d2b0d9330ad1a49e176b47d6cd3d